DFC.Digital/DFC.Digital.Service.CourseSearch/Converters/MessageConverter.cs

```diff
@@ -68,7 +68,7 @@ internal static IEnumerable<Course> ConvertToSearchCourse(this CourseListOutput
             return result ?? Enumerable.Empty<Course>();
         }
 
-        internal static CourseDetails ConvertToCourseDetails(this CourseDetailOutput apiResult)
+        internal static CourseDetails ConvertToCourseDetails(this CourseDetailOutput apiResult, string referralUrl)
         {
             var apiCourseDetail = apiResult.CourseDetails?.FirstOrDefault();
```

Comment:
FirstOrDefault is ever going to return the first venue result, this implementation is incorrect.